README.md

```diff
@@ -173,10 +173,10 @@ And invoked via `appInsights.<METHOD_NAME>`
 | ----------------------------|------------------------------------------------------------------------------------------------------------|
 | setDistributedTracingMode   | AI only tracing mode is no longer supported. Migrate to using W3C_AND_AI tracing mode. |
 | setAutoCollectHeartbeat     | Heartbeat is not supported in either the distro or the shim.|
-| enableWebInstrumenatation   | WebInstrumentation is not supported in either the distro or the shim. |
+| enableWebInstrumenatation   | WebInstrumentation is not supported in the shim, but is available in the distro as `enableBrowserSdkLoader` |
```

Comment:
Distro is not used in supported configuration and not explained anywhere in this readme, better to use  "Azure Monitor OpenTelemetry", you can also support old config in shim and change it to correct new value like other configs here

Reply:
I'll update that language. And I'll create a task for updating the shim to use the browser SDK loader in the distro once it's released.